corehq/apps/locations/templates/locations/manage/locations.html

```diff
@@ -1,163 +1,104 @@
+{% endblock %}
+
+{% block stylesheets %}{{ block.super }}
+    <link rel="stylesheet" href="{% new_static 'locations/style/manage.css' %}">
 {% endblock %}
 
 {% block js-inline %}{{ block.super }}
-<script>
-
-var LOAD_LOCS_URL = '{{ api_root }}',
-    NEW_LOC_URL = '{% url "create_location" domain %}';
-
-var locs = {{ locations|JSON }};
-var can_edit_root = {{ can_edit_root|yesno:"true,false" }};
-var selected_loc = {{ selected_id|JSON }};
-var hierarchy = {{ hierarchy|JSON }};
-var show_inactive = {{ show_inactive|yesno:"true,false" }};
-
-function loc_edit_url(loc_id) {
-    var template = '{% url "edit_location" domain "-locid-" %}';
-    return template.replace('-locid-', loc_id);
-}
-
-function loc_archive_url(loc_id) {
-    var template = '{% url "archive_location" domain "-locid-" %}';
-    return template.replace('-locid-', loc_id);
-}
-
-function loc_unarchive_url(loc_id) {
-    var template = '{% url "unarchive_location" domain "-locid-" %}';
-    return template.replace('-locid-', loc_id);
-}
-
-function loc_delete_url(loc_id) {
-    var template = '{% url "delete_location" domain "-locid-" %}';
-    return template.replace('-locid-', loc_id);
-}
-
-function loc_descendant_url(loc_id) {
-    var template = '{% url "location_descendants_count" domain "-locid-" %}';
-    return template.replace('-locid-', loc_id);
-}
-
-var archive_success_message = _.template(
-    '{% trans "You have successfully archived the location <%=name%>" %}'
-);
-
-var delete_success_message = _.template(
-    '{% trans "You have successfully deleted the location <%=name%> and all of its child locations" %}'
-);
-
-var delete_error_message = '{% trans "An error occurred while deleting your location. If the problem persists, please report an issue" %}'
-
-archive_loc = function(button, name, loc_id) {
-    var archive_location_modal = $('#archive-location-modal')[0];
-
-    function archive_fn() {
-        $(button).disableButton();
-        $.ajax({
-            type: 'POST',
-            url: loc_archive_url(loc_id),
-            dataType: 'json',
-            error: 'error',
-            success: function (response) {
-                alert_user(archive_success_message({"name": name}), "success");
-                remove_elements_after_action(button);
-            }
-        });
-        $(archive_location_modal).modal('hide');
+    <script>
+        var LOAD_LOCS_URL = '{{ api_root }}',
+            NEW_LOC_URL = '{% url "create_location" domain %}';
+
+        var locs = {{ locations|JSON }};
+        var can_edit_root = {{ can_edit_root|yesno:"true,false" }};
+        var selected_loc = {{ selected_id|JSON }};
+        var hierarchy = {{ hierarchy|JSON }};
+        var show_inactive = {{ show_inactive|yesno:"true,false" }};
+
+        function loc_edit_url(loc_id) {
+            var template = '{% url "edit_location" domain "-locid-" %}';
+            return template.replace('-locid-', loc_id);
+        }
     }
```

Comment:
I tried running this locally and got a failure on this extra `}`, I think it just needs to be removed.